lib/dal/src/attribute/value.rs

```diff
@@ -2271,29 +2271,35 @@ impl AttributeValue {
         ctx: &DalContext,
         subscriber_av_id: AttributeValueId,
         subscriptions: Vec<ValueSubscription>,
+        func_id: Option<FuncId>,
     ) -> AttributeValueResult<()> {
-        // Pick the prototype for the function based on prop type: if it's Array, use
-        // si:normalizeToArray, otherwise use si:identity
-        let func = match Self::prop(ctx, subscriber_av_id).await?.kind {
-            PropKind::Array => IntrinsicFunc::NormalizeToArray,
-            kind @ PropKind::Boolean
-            | kind @ PropKind::Integer
-            | kind @ PropKind::Json
-            | kind @ PropKind::Map
-            | kind @ PropKind::Object
-            | kind @ PropKind::String
-            | kind @ PropKind::Float => {
-                if subscriptions.len() != 1 {
-                    return Err(AttributeValueError::SingleValueMustHaveOneSubscription(
-                        subscriber_av_id,
-                        kind,
-                        subscriptions.len(),
-                    ));
+        let func_id = if let Some(id) = func_id {
+            id
+        } else {
+            // Pick the prototype for the function based on prop type: if it's Array, use
+            // si:normalizeToArray, otherwise use si:identity
+            let intrinsic_func = match Self::prop(ctx, subscriber_av_id).await?.kind {
+                PropKind::Array => IntrinsicFunc::NormalizeToArray,
```

Comment:
We're moving to a model where it's si:identity for everything, right? I assume this is just a stepping stone and we'll use si:identity in a next turn?